README.md

```diff
@@ -0,0 +1,112 @@
+Inheritance Shares Calculator
```

Comment:
Use proper heading and formatting styles provided by the markdown format. For details: https://docs.github.com/de/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

Reply:
got it

Reply:
got it